src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -3,45 +3,63 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { faExchangeAlt, faFolder, faHome, faQuestionCircle, faUsers, faChartGantt } from '@fortawesome/free-solid-svg-icons';
 import { routes } from '../../utils/routes';
-import { LinkItem } from '../../utils/types';
+import { MUILinkItem } from '../../utils/types';
 import NavPageLinks from './NavPageLinks';
 import styles from '../../stylesheets/layouts/sidebar/sidebar.module.css';
 import { Typography } from '@mui/material';
+// Material UI Icons
+import HomeIcon from '@mui/icons-material/Home';
+import BarChartIcon from '@mui/icons-material/BarChart';
+import FolderIcon from '@mui/icons-material/Folder';
+import SyncAltIcon from '@mui/icons-material/SyncAlt';
+import GroupIcon from '@mui/icons-material/Group';
+import HomeRepairServiceIcon from '@mui/icons-material/HomeRepairService';
+import QuestionMarkIcon from '@mui/icons-material/QuestionMark';
+import { useAuth } from '../../hooks/auth.hooks';
 
 const Sidebar: React.FC = () => {
-  const linkItems: LinkItem[] = [
+  const auth = useAuth();
+
+  const linkItems: MUILinkItem[] = [
     {
       name: 'Home',
-      icon: faHome,
+      icon: HomeIcon,
       route: routes.HOME
     },
     {
       name: 'Gantt',
-      icon: faChartGantt,
+      icon: BarChartIcon,
       route: routes.GANTT
     },
     {
       name: 'Projects',
-      icon: faFolder,
+      icon: FolderIcon,
       route: routes.PROJECTS
     },
     {
       name: 'Change Requests',
-      icon: faExchangeAlt,
+      icon: SyncAltIcon,
       route: routes.CHANGE_REQUESTS
     },
     {
       name: 'Teams',
-      icon: faUsers,
+      icon: GroupIcon,
       route: routes.TEAMS
     }
   ];
 
+  if (auth.user?.role === 'ADMIN' || auth.user?.role === 'APP_ADMIN') {
```

Comment:
we actually are moving admin tools to not be here anymore so we don't need this section at all anymore